docs/data/styles.json

```diff
@@ -48,17 +48,17 @@
                 "items": [
                     {
                       "title": "Sizes",
-                      "url": "",
+                      "url": "#spacing",
```

Comment:
Just a question from my side for understanding: how the "url": "#spacing" is linked to h2 with id="spacing-sizes" ?